src/main/java/com/revature/backend/model/Analysis_Item.java

```diff
@@ -0,0 +1,116 @@
+package com.revature.backend.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "analysisItems")
+public class Analysis_Item {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+
+    @Column(name = "content")
+    private String content;
+
+    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @JoinColumn(name = "swot_analysis_id")
+    private SWOT_Analysis SWOT_analysis_id;
+
+    @Column(name = "type")
+    private Analysis_Type type;
+
+    public Analysis_Item() {
+        // TODO Auto-generated constructor stub
+    }
+
+    public Analysis_Item(String content, SWOT_Analysis SWOT_analysis_id, Analysis_Type type) {
+        super();
+        this.content = content;
+        this.SWOT_analysis_id = SWOT_analysis_id;
+        this.type = type;
+    }
+
+    public Analysis_Item(int id, String content, SWOT_Analysis SWOT_analysis_id, Analysis_Type type) {
+        super();
+        this.id = id;
+        this.content = content;
+        this.SWOT_analysis_id = SWOT_analysis_id;
+        this.type = type;
+    }
+
+    public SWOT_Analysis getSwot_analysis_id() {
+        return SWOT_analysis_id;
+    }
+
+    public void setSwot_analysis_id(SWOT_Analysis SWOT_analysis_id) {
+        this.SWOT_analysis_id = SWOT_analysis_id;
+    }
+
+    public int getId() {
+        return this.id;
+    }
+
+    public String getContent() {
+        return this.content;
+    }
+
+    public Analysis_Type getType() {
+        return this.type;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public void setContent(String content) {
+        this.content = content;
+    }
+
+    public void setType(Analysis_Type type) {
+        this.type = type;
+    }
+
+    public boolean equals(final Object o) {
+        if (o == this) return true;
+        if (!(o instanceof Analysis_Item)) return false;
+        final Analysis_Item other = (Analysis_Item) o;
+        if (!other.canEqual((Object) this)) return false;
+        if (this.getId() != other.getId()) return false;
+        final Object this$content = this.getContent();
+        final Object other$content = other.getContent();
+        if (this$content == null ? other$content != null : !this$content.equals(other$content)) return false;
```

Comment:
Is this supposed to be `this$content == null`?

Reply:
That's a remnant of delombaking, I'll see what they were meaning to do